system/iptables.py

```diff
@@ -266,6 +266,12 @@
     description:
       - "Specifies the error packet type to return while rejecting."
     required: false
+  setup:
```

Comment:
setup does not sound right to me (yep, i'm bikeshedding), it might make more sense to have `postition=apend|prepend|<line number>`